bricks/presentational-bricks/src/brick-table/index.tsx

```diff
@@ -785,6 +786,17 @@ export class BrickTableElement extends UpdatingElement {
   })
   frontSearchQuery = "";
 
+  /**
+   * @kind boolean
+   * @required false
+   * @default false
+   * @description 是否精确搜索
+   */
+  @property({
+    attribute: false,
+  })
+  exactSearch = false;
```

Comment:
```suggestion
  @property({
    type: Boolean,
  })
  exactSearch: boolean;
```

对于简单类型（String|Boolean|Number）应使用对应的 `type` 而不是盲目设置 `attribute: false`，这样可以激活自定义元素的 attribute 与 property 的自动对应。